SKILL.tsv

```diff
@@ -5359,29 +5359,29 @@ SKILL_20151022_005358
 SKILL_20151022_005359	
 SKILL_20151022_005360	
 SKILL_20151022_005361	$Attaque Composee
-SKILL_20151022_005362	
+SKILL_20151022_005362	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Stab an enemy in front of you.
 SKILL_20151022_005363	$Lunge
-SKILL_20151022_005364	
-SKILL_20151022_005365	
+SKILL_20151022_005364	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Lower your stance and reach out to attack an enemy. Your evasion rate increases and the enemy becomes more vulnerable to [Slash] attacks.
+SKILL_20151022_005365	$Attack #{SkillAtkAdd}#{nl}Attribute Damage #{CaptionRatio}#%{nl}Evasion Rate #{CaptionRatio2}#%{nl}Duration #{CaptionTime}# seconds
```

Comment:
Attack #{SkillAtkAdd}#{nl}Attribute Damage #{CaptionRatio}#%{nl}Evasion Rate #{CaptionRatio2}#%{nl}Duration #{CaptionTime}# seconds

to

Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Evasion Rate: +#{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds